spec/cobra_commander/cli_spec.rb

```diff
@@ -291,12 +291,30 @@
       expect(last_command_output).to match(/Component node_manifeast not found, maybe node_manifest, one of "cobra ls"/)
     end
 
+    it "suggests with DidYouMean when one of the list does not exist", if: RUBY_2_7 do
+      run_command_and_stop("cobra ls -a #{fixture_app} node_manifeast,a,b", fail_on_error: false)
+
+      expect(last_command_output).to match(/Component node_manifeast not found, maybe node_manifest, one of "cobra ls"/)
```

Comment:
```suggestion
      expect(last_command_output).to match(/Component node_manifest not found, maybe node_manifest, one of "cobra ls"/)
```

Reply:
The typo is intended. This spec checks if cobra can suggest the correct name.